include/aspect/initial_temperature/lithosphere_mask.h

```diff
@@ -0,0 +1,128 @@
+/*
+  Copyright (C) 2011 - 2018 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_initial_temperature_lithosphere_mask_h
+#define _aspect_initial_temperature_lithosphere_mask_h
+
+#include <aspect/initial_temperature/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/utilities.h>
+
+
+namespace aspect
+{
+  namespace InitialTemperature
+  {
+    using namespace dealii;
+
+    /**
+     * A class that implements a constant reference temperature
+     * above a specified depth and nans below the specified depth.
+     *
+     * @ingroup InitialTemperatures
+     */
+    template <int dim>
+    class LithosphereMask : public Interface<dim>, public SimulatorAccess<dim>
+    {
+      public:
+        /**
+         * Empty Constructor.
+         */
+        LithosphereMask ();
+
+        /**
+         * Initialization function. This function is called once at the
+         * beginning of the program. Checks preconditions.
+         */
+        void
+        initialize ();
+
+        /**
+         * Directory in which the LAB depth file is present.
+         */
+        std::string data_directory;
+
+        /**
+         * File name of the LAB depth file.
+         */
+        std::string LAB_file_name;
+
+        /**
+         * Return LAB depth as a function of position (latitude and longitude). For the
+         * current class, this function returns value from the text files.
+         */
+        double
+        ascii_lab (const Point<2> &position) const;
```

Comment:
does this function need to `public`? I.e. do you need to call it from outside the .cc file?